process-engine/bpmn-studio
# Move Property Panel to Plugin

## What did you change?

This branch will move the Property Panel to an aurelia plugin.

This PR waits for https://github.com/process-engine/bpmn-studio_property-panel/pull/1 !

## How can others test the changes?

You can test theese changes by:

- checkout this branch AND the branch of https://github.com/process-engine/bpmn-studio_property-panel/pull/1
- run `npm install`, `npm run build` and `npm link` in the plugin
- run `npm install`, `npm link bpmn-studio_property-panel` and `au run` here
- Open the editor and see how the property panel is still working.

Closes #128 

## PR-Checklist

Please check the boxes in this list after submitting your PR:
